data/BLUE/data.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "BlueSwap",
+  "symbol": "BLUE",
+  "decimals": 18,
+  "description": "Swap & Liquidity Pool Token for BlueSwap DEX",
+  "website": "https://blueswap.app",
+  "twitter": "@blueswapbase",
+  "tokens": {
```

Comment:
does this token exist on Ethereum? If so please add the appropriate section here.

Reply:
It does not, does it need to exist on Ethereum to be in the list? It's a Base only token.

Reply:
No it does not, but if it's a native L2 token that can't be bridged you'll want to add the "nobridge": true directive to your data.json.

Reply:
Thank you, done I added that to the request.